submodules/wizard/views/step-appRestrictions.html

```diff
@@ -0,0 +1,86 @@
+<div id="step_app_restrictions" class="wizard-step">
+{{#with i18n.accountsApp.wizard.steps.appRestrictions as |appRestrictionsI18n|}}
+	<div class="step-title">
+		{{appRestrictionsI18n.title}}
+	</div>
+	<div class="step-body">
+		<form id="form_app_restrictions" name="form_app_restrictions" action="javascript:void(0);">
+			<div class="section-body flex-row-container">
+				<div class="flex-item-2">
+					<div class="flex-row-container">
+						<div class="flex-item-1">
+							{{appRestrictionsI18n.accessLevel.description}}
+						</div>
+					</div>
+					<div class="flex-row-container">
+						<div class="flex-item-1 flex-container access-level-toggle">
+							<div id="access_level" class="btn-group">
+								<input type="radio" name="accessLevel" id="access_level_full" class="radio-button input-hidden" value="full"{{#compare @root.data.accessLevel "===" "full"}} checked{{/compare}}/>
+								<label for="access_level_full" class="monster-button monster-button-fit">
+									{{appRestrictionsI18n.accessLevel.types.full}}
+								</label>
+								<input type="radio" name="accessLevel" id="access_level_restricted" class="radio-button input-hidden" value="restricted"{{#compare @root.data.accessLevel "!==" "full"}} checked{{/compare}}/>
+								<label for="access_level_restricted" class="monster-button monster-button-fit">
+									{{appRestrictionsI18n.accessLevel.types.restricted}}
+								</label>
+							</div>
+							<div>
+								<span class="help-tip help-tip-button" data-title="{{appRestrictionsI18n.accessLevel.help}}" data-toggle="tooltip">
+									{{telicon "question--circle" class="iconography-small iconography-help"}}
+								</span>
+							</div>
+						</div>
+					</div>
+				</div>
+				<div class="flex-item-2">
+					{{#monsterPanelText appRestrictionsI18n.accessLevel.important 'info'}}
+						<p>{{appRestrictionsI18n.accessLevel.notice}}</p>
+					{{/monsterPanelText}}
+				</div>
+			</div>
+			<div id="section_allowed_apps" {{#compare @root.data.accessLevel "===" "full"}}style="display: none"{{else}}class="visible"{{/compare}}>
+				<div class="section-title">
+					{{appRestrictionsI18n.apps.title}}
+				</div>
+				<div class="section-body">
+					<div class="flex-row-container">
+						<div class="flex-item-2">
+							{{appRestrictionsI18n.apps.description}}
+						</div>
+					</div>
+					<div class="flex-row-container app-list">
+					{{#each @root.apps}}
+						<div id="app_{{id}}" class="flex-item-3 flex-container app-item{{#ifInArray id @root.data.allowedAppIds}} visible"{{else}}" style="display:none"{{/ifInArray}} data-id="{{id}}">
+							<div class="wizard-card">
+								<div class="card-section title flex-container">
+									<div class="app-icon {{extraCssClass}}" style="{{#if icon}}background-image:url({{icon}}){{else}}background-color:#606069{{/if}}"></div>
+									<div class="app-name">
+										{{label}}
+									</div>
+									<label class="app-remove" href="javascript:void(0);">
```

Comment:
`href` is not a valid attribute for `label`.

Also, semantically this tag should a link rather than a label.

Reply:
Yes, I forgot about the `href` attribute :sweat_smile: I used a label because it contained the hidden checkbox input, which kind of toggles the app selection. But for the user does not look like a toggle, so I think that the link is better.